.github/workflows/alex_reviewdog.yml

```diff
@@ -25,7 +25,6 @@ on:
     types: [ opened, synchronize ]
     branches:
       - main
-      - integration
```

Comment:
Why has this branch been removed? All of our PRs are based against `integration`.

Reply:
@venkuppu-chn, please address the comment as raised